test/cpp/interop/xds_interop_client.cc

```diff
@@ -214,21 +290,47 @@ class LoadBalancerStatsServiceImpl : public LoadBalancerStatsService::Service {
   }
 };
 
-void RunTestLoop(const std::string& server,
-                 std::chrono::duration<double> duration_per_query) {
+void RunTestLoop(std::chrono::duration<double> duration_per_query) {
+  std::vector<std::string> rpc_types = absl::StrSplit(FLAGS_rpc, ',');
+  // Store Metadata like
+  // "EmptyCall:key1:value1,UnaryCall:key1:value1,UnaryCall:key2:value2" into a
+  // map where the key is the RPC type and value is a vector of key:value pairs.
+  // {EmptyCall, [{key1,value1}],
+  //  UnaryCall, [{key1,value1}, {key2,value2}]}
+  std::vector<std::string> rpc_metadata = absl::StrSplit(FLAGS_metadata, ',');
+  std::map<std::string, std::vector<std::pair<std::string, std::string>>>
+      metadata_map;
+  for (auto& data : rpc_metadata) {
+    std::vector<std::string> metadata = absl::StrSplit(data, ':');
+    GPR_ASSERT(metadata.size() == 3);
+    metadata_map[metadata[0]].push_back(std::pair<std::string, std::string>(
```

Comment:
Instead of saying `std::pair<std::string, std::string>(foo, bar)`, I think you can just say `{foo, bar}`.